boards/nxp/frdm_mcxaxx6/frdm_mcxa366.dts

```diff
@@ -8,7 +8,7 @@
 
 #include <nxp/nxp_mcxa366.dtsi>
 #include "frdm_mcxa366-pinctrl.dtsi"
-#include "../common/frdm_mcxa266_a346_a366_common.dtsi"
```

Comment:
why is a whole being added in one commit then being removed in another commit, fix it so that only one board gets added without removing it